src/LoadableModules.cc

```diff
@@ -8,29 +8,30 @@
 
 #include "squid.h"
 #include "debug/Messages.h"
-#include "debug/Stream.h"
 #include "LoadableModule.h"
 #include "LoadableModules.h"
-#include "wordlist.h"
+#include "sbuf/List.h"
 
 static void
-LoadModule(const char *fname)
+LoadModule(const SBuf &fname)
 {
     debugs(1, DBG_IMPORTANT, "Loading Squid module from '" << fname << "'");
 
-    LoadableModule *m = new LoadableModule(fname);
+    auto *m = new LoadableModule(fname);
```

Comment:
This line did not have to be changed AFAICT, but if this PR changes it, let's improve its const-correctness.

```suggestion
    const auto m = new LoadableModule(fname);
```

Reply:
done in [fd858aa](https://github.com/squid-cache/squid/pull/1738/commits/fd858aaa13035510319477ae023f7064982d4576)